### ISSUE TITLE
Learning Rate Scheduler

### ISSUE THREAD

**Issue by aminzabardast** [aminzabardast/MastersThesis#58]
Implement a `Keras` [`Callback`](https://keras.io/callbacks/) to adapt Learning Rate.
